chandra_aca/drift.py

```diff
@@ -0,0 +1,175 @@
+"""
+Function(s) related to ACA alignment.
+
+In particular compute the dynamical pointing offset required to achieve
+a desired zero-offset target aimpoint.
+
+A key element of this module is the fitting analysis here:
+https://github.com/sot/aimpoint_mon/blob/master/fit_aimpoint_drift.ipynb
+"""
+
+from Chandra.Time import DateTime
+
+import numpy as np
+
+# Capture best fit model parameters for ACA drift model.
+# https://github.com/sot/aimpoint_mon/blob/8259a23/fit_aimpoint_drift.ipynb
+# These are used below to instantiate corresponding AcaDriftModel objects.
+
+DRIFT_Y_PARS = dict(scale=2.183,  # drift per degF (NOT degC as elsewhere in this module)
+                    offset=-5.606,
+                    trend=-1.363,
+                    jumps=(('2015:006', -4.234),
+                           ('2015:265', -4.571),
+                           ('2016:064', -2.027)),
+                    year0=2016.0)
+
+DRIFT_Z_PARS = dict(scale=1.067,
+                    offset=-14.374,
+                    trend=-0.407,
+                    jumps=(('2015:006', -1.8154),
+                           ('2015:265', -0.3033),
+                           ('2016:064', -1.1105)),
+                    year0=2016.0)
+
+# Define transform from aspect solution DY, DZ (mm) to CHIPX, CHIPY for
+# nominal target position.  This uses the transform:
+# CHIPX, CHIPY = c0 + cyz * [DY, DZ]  (dot product)
+# These values come from the aimpoint_mon/asol_to_chip_transforms notebook:
+# https://github.com/sot/aimpoint_mon/blob/cebe5010/asol_to_chip_transforms.ipynb
+ASOL_TO_CHIP = {('ACIS-I', 0): {'c0': [1100.806, 1110.299],
+                                'cyz': [[0.001, 41.742], [-41.742, 0.105]]},
+                ('ACIS-I', 1): {'c0': [-76.208, 962.413],
```

Comment:
Late to the party, but would tdetx, tdety have worked better than chipx, chipy here?  Not sure.  I also don't completely understand the asol_to_chip_transforms notebook 3 points and how they work with the angles of the ACIS-I chips.